src/evict.c

```diff
@@ -80,7 +80,7 @@ unsigned int getLRUClock(void) {
 unsigned int LRU_CLOCK(void) {
     unsigned int lruclock;
     if (1000/server.hz <= LRU_CLOCK_RESOLUTION) {
-        atomicGet(server.lruclock,lruclock);
```

Comment:
i suppose you did that because we are now sure (even more than before) that all the access to server.lruclock are from the main thread.

in that case, let's drop all usages of `atomicSet` / `atomicGet` when accessing this var, and also remove the use of `redisAtomic` when declaring it.

additinally, either change this line to `lruclock = server.lruclock;` (instead of `return`), or drop the local var and fix the line in the `else` too.

Reply:
thank you and done